.babelrc.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-react',
+      {
+        runtime: 'automatic',
+        importSource: 'preact',
+      },
+    ],
+    [
+      '@babel/preset-env',
+      {
+        bugfixes: true,
+        modules: false, // Produce ES module output
+        targets: {
+          chrome: '70',
+          firefox: '70',
+          safari: '11.1',
+          edge: '79',
```

Comment:
I think it is better to take this out and rely on `browserslist` in package.json, to give consistency to all our projects.

Maybe the browsers should be similar to the list in `lms` not the `client`? 
`"browserslist": "chrome 57, edge 17, firefox 53, safari 10.1",`

Reply:
> I think it is better to take this out and rely on browserslist in package.json, to give consistency to all our projects.

I agree. I'll do that in this PR.

> Maybe the browsers should be similar to the list in lms not the client?

The effective browser requirements for the lms app is the _maximum_ of the lms and client's code, since the lms app isn't useful without the client. I intend to update the client and lms app to have the same baseline as this.

It also occurred to me that since Edge is now based on Chrome, there shouldn't be a need to specify the Edge version any more. We can just treat it as being covered by Chrome.